cmd/osm-bootstrap/crds/config_multicluster_service.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     plural: multiclusterservices
     singular: multiclusterservice
   versions:
-    - name: v1alpha1
+    - name: v1alpha2
```

Comment:
Why are we updating the multicluster crd as a part of this PR ?

Reply:
This change migrates the controller to use `v1alpha2` of the config.openservicemesh.io API group, which also includes multicluster. v1alpha1 API will be deprecated later on (already marked as deprecated).